docs/en/observability/synthetics-role-write.asciidoc

```diff
@@ -0,0 +1,145 @@
+
+* *General write access*: For most users, you can use <<synthetics-write-privileges-general>>,
+which grants write access to all {kib} apps and requires little configuration.
+* *Limited write access*: If you want to limit write access to the {synthetics-app} only,
+you can use <<synthetics-write-privileges-limited>>, which requires additional configuration.
+
+[discrete]
+[[synthetics-write-privileges-general]]
+= General write access
+
+Create a *writer role*, called something like `synthetics_writer`:
+
+. Start with the `editor` {ref}/built-in-roles.html[built-in role].
+This role grants full access to all features in {kib} (including the {observability} solution)
+and read-only access to data indices.
++
+NOTE: The `editor` {ref}/built-in-roles.html[built-in role] will grant write access
+to _all_ {kib} apps. If you want to limit write access to the {synthetics-app} only,
+refer to <<synthetics-write-privileges-limited>>.
+
+. _If the user should have permission to create, modify, and delete project monitors_,
+they will need an API key that can be used to `push` monitors.
+To create API keys, the user will also need _at least one_ of the following privileges in addition to the
+privileges included in the `editor` built-in role:
++
+[options="header"]
+|====
+|Type | Privilege | Purpose
+
+| {ref}/security-privileges.html#privileges-list-cluster[Cluster]
+| `manage_own_api_key`
+| Allows access to all security-related operations on {es} API keys that are owned by the current authenticated user.
+
+| {ref}/security-privileges.html#privileges-list-cluster[Cluster]
+| `manage_security`
+| Allows access to all security-related operations such as CRUD operations on users and roles and cache clearing.
+
+| {ref}/security-privileges.html#privileges-list-cluster[Cluster]
+| `manage_api_key`
+| Allows access to all security-related operations on {es} API keys.
+
+|====
+
+[discrete]
+[[synthetics-write-privileges-limited]]
+= Limited write access
+
+If you want to limit write access to the {synthetics-app} only, do _not_ use the
+`editor` {ref}/built-in-roles.html[built-in role].
+
+Instead to you can create a writer role, called something like `synthetics_writer_limited`,
+and start by granting the following privileges:
+
+[options="header"]
+|====
+|Type | Privilege | Purpose
+
+| {ref}/security-privileges.html#privileges-list-indices[Index]
+| `synthetics-*`: `read`
+| Read-only access to synthetics indices.
+
+| {ref}/security-privileges.html#privileges-list-indices[Index]
+| `.alerts-observability.uptime.alerts-*`: `read`
+| Read-only access to synthetics alert indices.
+|====
+
+Additional privileges will depend on the factors outlined below.
+
+[discrete]
+== Q: Where are monitors hosted?
+
+[discrete]
+=== A: Elastic's global managed infrastructure
+
+Include the following privileges:
+
+[options="header"]
+|====
+|Type | Privilege
+
+| {kibana-ref}/kibana-privileges.html[Kibana]
+| `Uptime/Synthetics`: `All`
```

Comment:
I think this should be a base privilege for both the limited write and read user, rather than included in the questions like this.